plugins/aladino/actions/review_test.go

```diff
@@ -0,0 +1,151 @@
+	wantReviewEvent := "COMMENT"
+	wantReviewComment := "test comment"
+	var (
+		gotReviewEvent   string
+		gotReviewComment string
+	)
+	mockedEnv := aladino.MockDefaultEnv(
+		t,
+		[]mock.MockBackendOption{
+			mock.WithRequestMatchHandler(
+				mock.PostReposPullsReviewsByOwnerByRepoByPullNumber,
+				http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+					rawBody, _ := ioutil.ReadAll(r.Body)
+					body := ReviewRequestPostBody{}
+
+					json.Unmarshal(rawBody, &body)
+
+					gotReviewEvent = body.Event
+					gotReviewComment = body.Body
+				}),
+			),
+		},
+		nil,
+		aladino.MockBuiltIns(),
+		nil,
+	)
+
+	args := []aladino.Value{aladino.BuildStringValue(wantReviewEvent), aladino.BuildStringValue(wantReviewComment)}
+	err := review(mockedEnv, args)
+
+	assert.Nil(t, err)
+	assert.Equal(t, wantReviewEvent, gotReviewEvent)
+	assert.Equal(t, wantReviewComment, gotReviewComment)
+}
+
+func TestReview_WhenReviewEventIsRequestChanges(t *testing.T) {
+	wantReviewEvent := "REQUEST_CHANGES"
+	wantReviewComment := "test comment"
+	var (
+		gotReviewEvent   string
+		gotReviewComment string
+	)
+	mockedEnv := aladino.MockDefaultEnv(
+		t,
+		[]mock.MockBackendOption{
+			mock.WithRequestMatchHandler(
+				mock.PostReposPullsReviewsByOwnerByRepoByPullNumber,
+				http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+					rawBody, _ := ioutil.ReadAll(r.Body)
+					body := ReviewRequestPostBody{}
+
+					json.Unmarshal(rawBody, &body)
+
+					gotReviewEvent = body.Event
+					gotReviewComment = body.Body
+				}),
+			),
+		},
+		nil,
+		aladino.MockBuiltIns(),
+		nil,
+	)
+
+	args := []aladino.Value{aladino.BuildStringValue(wantReviewEvent), aladino.BuildStringValue(wantReviewComment)}
+	err := review(mockedEnv, args)
+
+	assert.Nil(t, err)
+	assert.Equal(t, wantReviewEvent, gotReviewEvent)
+	assert.Equal(t, wantReviewComment, gotReviewComment)
+}
+
+func TestReview_WhenReviewEventIsApprove(t *testing.T) {
+	wantReviewEvent := "APPROVE"
+	wantReviewComment := "test comment"
+	var (
+		gotReviewEvent   string
+		gotReviewComment string
+	)
+	mockedEnv := aladino.MockDefaultEnv(
+		t,
+		[]mock.MockBackendOption{
+			mock.WithRequestMatchHandler(
+				mock.PostReposPullsReviewsByOwnerByRepoByPullNumber,
+				http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+					rawBody, _ := ioutil.ReadAll(r.Body)
+					body := ReviewRequestPostBody{}
+
+					json.Unmarshal(rawBody, &body)
+
+					gotReviewEvent = body.Event
+					gotReviewComment = body.Body
+				}),
+			),
+		},
+		nil,
+		aladino.MockBuiltIns(),
+		nil,
+	)
+
+	args := []aladino.Value{aladino.BuildStringValue(wantReviewEvent), aladino.BuildStringValue(wantReviewComment)}
+	err := review(mockedEnv, args)
+
+	assert.Nil(t, err)
+	assert.Equal(t, wantReviewEvent, gotReviewEvent)
+	assert.Equal(t, wantReviewComment, gotReviewComment)
+}
+
+func TestReview_WhenReviewEventIsRequestChangesWithoutComment(t *testing.T) {
```

Comment:
We should also consider the COMMENT event.
```suggestion
func TestReview_WhenBodyIsEmpty(t *testing.T) {
	tests := map[string]struct {
		inputReviewEvent string
		wantErr          string
	}{
		"when review event is APPROVE": {
			inputReviewEvent: "APPROVE",
			wantErr:          "",
		},
		"when review event is REQUEST_CHANGES": {
			inputReviewEvent: "REQUEST_CHANGES",
			wantErr:          "review: comment required in REQUEST_CHANGES event",
		},
		"when review event is COMMENT": {
			inputReviewEvent: "COMMENT",
			wantErr:          "review: comment required in COMMENT event",
		},
	}

	for name, test := range tests {
		t.Run(name, func(t *testing.T) {
			mockedEnv := aladino.MockDefaultEnv(t, nil, nil, aladino.MockBuiltIns(), nil)

			args := []aladino.Value{aladino.BuildStringValue(test.inputReviewEvent), aladino.BuildStringValue("")}
			gotErr := review(mockedEnv, args)

			if gotErr != nil && gotErr.Error() != test.wantErr {
				assert.FailNow(t, "review() error = %v, wantErr %v", gotErr, test.wantErr)
			}
		})
	}
}
```